test/functional/framework.go

```diff
@@ -106,12 +106,12 @@ func NewFluentdFunctionalFrameworkUsing(t *client.Test, fnClose func(), verbosit
 	return framework
 }
 
-func NewCRIOLogMessage(timestamp, message string, partial bool) string {
+func (f *FluentdFunctionalFramework) NewCRIOLogMessage(timestamp, message string, partial bool) string {
```

Comment:
No need to make this part of the framework as it does not appear to need anything from `f` to function

Reply:
ok